src/Nethermind/Nethermind.Serialization.Rlp/RlpBehaviors.cs

```diff
@@ -29,6 +29,8 @@ public enum RlpBehaviors
         Eip658Receipts = 8,
         AllowUnsigned = 16,
         ForTxRoot = 32,
+#pragma warning disable 0618
```

Comment:
Do you prefer this style or the style from above for `#pragma warning disable/restore`?